azure-pipelines.yml

```diff
@@ -43,6 +41,7 @@ stages:
           - task: BatchScript@1
             displayName: 'Enforce clang-format'
             timeoutInMinutes: 60
+            condition: succeededOrFailed()

```

Comment:
Won't this run the enforcement script even if building the (necessary) support tools (the previous step) fails?

Reply:
Yes, but it'll just immediately fail due to not finding the tool. Specifying dependencies requires to use jobs, which means we have to bother with artifacts.

Reply:
So _this_ line could as well be the default (`succeeded()`); it's fine for this step to fail if the previous step failed.

Reply:
...or should we mark both of these steps `continueOnError`? It seems like that would be equivalent since there are no further steps.

Reply:
`continueOnError` does not make the build error, it turns those failures into warnings only

Reply:
That would certainly not be equivalent, then. Thanks for the clarifications.